official/recommendation/ncf_keras_benchmark.py

```diff
@@ -181,6 +181,13 @@ def benchmark_1_gpu_ctl_early_stop(self):
     FLAGS.early_stopping = True
     self._run_and_report_benchmark()
 
+  def benchmark_1_gpu_ctl_run_eagerly_early_stop(self):
```

Comment:
Do we also want to do a non dist strat run eagerly?

Reply:
Now I am confused again; and now I remember dist strat needs turned off.  I always look at this branch last when I am rushing to finish other stuff.  One more try coming up.  :-)

Reply:
Other than removing tests this is the last test I am adding period... probably....no seriously...maybe.